[2025-02-17T19:02:26.083+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: etl_pipeline.extract manual__2025-02-17T19:02:10.791572+00:00 [queued]>
[2025-02-17T19:02:26.094+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: etl_pipeline.extract manual__2025-02-17T19:02:10.791572+00:00 [queued]>
[2025-02-17T19:02:26.095+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-02-17T19:02:26.097+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2025-02-17T19:02:26.098+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-02-17T19:02:26.117+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): extract> on 2025-02-17 19:02:10.791572+00:00
[2025-02-17T19:02:26.121+0000] {standard_task_runner.py:55} INFO - Started process 255 to run task
[2025-02-17T19:02:26.125+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'extract', 'manual__2025-02-17T19:02:10.791572+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmploedarht']
[2025-02-17T19:02:26.127+0000] {standard_task_runner.py:83} INFO - Job 10: Subtask extract
[2025-02-17T19:02:26.145+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-17T19:02:26.208+0000] {task_command.py:388} INFO - Running <TaskInstance: etl_pipeline.extract manual__2025-02-17T19:02:10.791572+00:00 [running]> on host 06a0cbcfaed3
[2025-02-17T19:02:26.309+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_pipeline
AIRFLOW_CTX_TASK_ID=extract
AIRFLOW_CTX_EXECUTION_DATE=2025-02-17T19:02:10.791572+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-02-17T19:02:10.791572+00:00
[2025-02-17T19:02:26.310+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2025-02-17T19:02:26.311+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "üîç Iniciando extracci√≥n de datos..." && sleep 2 && echo "‚úÖ Extracci√≥n completada."']
[2025-02-17T19:02:26.319+0000] {subprocess.py:86} INFO - Output:
[2025-02-17T19:02:26.320+0000] {subprocess.py:93} INFO - üîç Iniciando extracci√≥n de datos...
[2025-02-17T19:02:28.324+0000] {subprocess.py:93} INFO - ‚úÖ Extracci√≥n completada.
[2025-02-17T19:02:28.325+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-02-17T19:02:28.370+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=extract, execution_date=20250217T190210, start_date=20250217T190226, end_date=20250217T190228
[2025-02-17T19:02:28.395+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2025-02-17T19:02:28.434+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
